x/gamm/keeper/swap.go

```diff
@@ -150,7 +150,7 @@ func (k Keeper) SwapExactAmountOut(
 	}
 
 	if tokenInAmount.GT(tokenInMaxAmount) {
-		return sdk.Int{}, sdk.Dec{}, sdkerrors.Wrapf(types.ErrLimitMinAmount, "%s token is larger than max amount", outRecord.Token.Denom)
+		return sdk.Int{}, sdk.Dec{}, sdkerrors.Wrapf(types.ErrLimitMaxAmount, "%s token is larger than max amount", outRecord.Token.Denom)
```

Comment:
I'm going to cherry-pick this into a separate PR, just so we can keep x/gamm up to date on main